src/views/Deposit/components/EnterAmount/components/index.ts

```diff
@@ -191,10 +191,10 @@ export const StyledMaxButton = styled.button`
         align-items: center;
         display: flex;
         fill: ${colors.yellow};
-        height: .8rem;
-        margin-left: .8rem;
+        height: 0.8rem;
```

Comment:
Let's stick to `px`s for non-text-based things ;)

Reply:
Fixed in 71b12631f095a0169dd86908bf04935aeec207ef